### DISCUSSION TITLE
Question about how to get R2* map

### DISCUSSION THREAD

**Discussion by Goodweatherz** [kschan0214/sepia#81] [Q&A]
Hi, I noticed that SEPIA has the function to generate R2 * map, but I couldn't find the relevant documentation. Could you tell me what files should I input to generate R2 * map?  Thank you so much!

**Comment by kschan0214** [comment]
Hi, unfortunately, I haven't had time to update the documentary website to include the instructions for R2* mapping yet. Which SEPIA version are you using? From v1.2.x onwards, R2* mapping is also supported in the same SEPIA GUI

![Screenshot 2024-01-26 at 12 47 37 PM](https://github.com/kschan0214/sepia/assets/39571502/06f4562c-3001-4de8-aa0d-0124a39b6ae0)

Similar to the QSM pipeline, a .m script should be generated automatically that contains all method parameters and can be run without initialising the GUI.
